arviz/plots/backends/bokeh/bpvplot.py

```diff
@@ -175,12 +176,12 @@ def plot_bpv(
                     obs_vals.mean(), 0, fill_color=color, line_color="black", size=markersize
                 )
 
-        if var_name != pp_var_name:
-            xlabel = "{} / {}".format(var_name, pp_var_name)
-        else:
-            xlabel = var_name
+        # if var_name != pp_var_name:
+        #     xlabel = "{} / {}".format(var_name, pp_var_name)
+        # else:
+        #     xlabel = var_name
         _title = Title()
-        _title.text = xlabel
+        _title.text = labeller.make_label_vert(var_name, sel, isel)
```

Comment:
Here matplotlib and bokeh behaved diferently. I have modified bokeh to match mpl, but we can also add a new method to the labellers to take the possible 2 var_names into account. cc @aloctavodia